package.json

```diff
@@ -35,7 +36,7 @@
     "expo-web-browser": "~13.0.3",
     "lucide-react": "^0.439.0",
     "lucide-react-native": "^0.439.0",
-    "nativewind": "^4.0.1",
+    "nativewind": "^4.0.36",
```

Comment:
Which styles weren't showing up out of interest? Does bumping this solve it?

Reply:
I was unable to set the border colour on the TouchableOpacity elements, and I found others with the same issue. As we discussed however TouchableOpacity is being deprecated anyway and we should probably switch to using Pressables at some point.

Reply:
hmmm, border colour should have been a thing from a long time ago. I'm sure it's fine to leave like this though.

Reply:
Maybe when Nativewind was building v4 they didn't test it thoroughly on those React Native elements that are being deprecated.